rust/lance-linalg/build.rs

```diff
@@ -35,4 +49,23 @@ fn main() {
             .flag("-Wpedantic")
             .compile("f16");
     }
+
+    if cfg!(all(target_os = "linux", feature = "avx512bf16")) {
+        // No enable bf16 kernels on sappphire rapids
```

Comment:
`No bf16 on sapphire rapid?` or `only bf16 kernel on sapphier rapid`?

Reply:
Whoa, I don't know what I was thinking there.
```suggestion
        // Enable avx512bf16 for bf16 kernels only on sappphire rapids
```